crates/types/src/serde.rs

```diff
@@ -73,15 +75,47 @@ pub fn parse_state_diffs(data: &[BigUint]) -> Vec<ContractUpdate> {
         });
     }
 
-    updates
+    let declared_classes_len = data[i].to_usize().unwrap();
+    let mut class_declaration_updates = Vec::new();
+    i += 1;
+    for _ in 0..declared_classes_len {
+        let class_hash = data[i].clone();
+        // Break if address undefined
+        if class_hash == BigUint::zero() {
+            println!("no class hash declared ser!!!");
```

Comment:
let's remove the println. also, if the length is > 0, then is this case an error?

Reply:
yes if length is > 0 and if the class hash is zero, then it would be an error